
# Timestap of the microcontroller, in millisecs since boot
uint32  timestamp_ms
# PWM
int32   Brake_controller_signal
 # DAC
float64   Throttle_controller_signal
# Encoder Incremental
float64   Encoder_Incremental
# ADC value for steer motor current sensor (Volts)
float64   Brake_ADC_current_sense
# ADC value for pedal feedback (Volts)
float64   Throttle_analog_feedback
